<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rock Paper Scissors</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(35, 37, 46);
color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="PlayerLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>156</width>
      <height>227</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ComputerLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>156</width>
      <height>227</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Player">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>You</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Computer">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Computer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ComputerPoints">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerPoints">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Rock">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Paper">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Paper</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Scissors">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>370</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Scissors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>vs</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="PlayerResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ComputerResult">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>330</y>
      <width>156</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
